### PR TITLE
Fix missing valid_attrs (issue 4529)

### PR DESCRIPTION
**Pull request by thisjulio**
Fix missing valid_attrs required when create application load balancer in terraform

related to #4529

### CODE REVIEW

File: tests/test_elbv2/test_elbv2.py [hunk 1]
```diff
@@ -1146,6 +1146,52 @@ def test_modify_load_balancer_attributes_idle_timeout():
...
+def test_modify_load_balancer_attributes_routing_http2_enabled():
+    response, _, _, _, _, client = create_load_balancer()
+    arn = response["LoadBalancers"][0]["LoadBalancerArn"]
+
+    client.modify_load_balancer_attributes(
+        LoadBalancerArn=arn,
+        Attributes=[{"Key": "routing.http2.enabled", "Value": "false"}],
+    )
+
+    # Check its 600 not 60
```
Comment (bblommers) [inline tests/test_elbv2/test_elbv2.py:1160]: This comment seems unrelated

File: tests/test_elbv2/test_elbv2.py [hunk 2]
```diff
@@ -1146,6 +1146,52 @@ def test_modify_load_balancer_attributes_idle_timeout():
...
+    arn = response["LoadBalancers"][0]["LoadBalancerArn"]
+
+    client.modify_load_balancer_attributes(
+        LoadBalancerArn=arn,
+        Attributes=[
+            {"Key": "routing.http.drop_invalid_header_fields.enabled", "Value": "false"}
+        ],
+    )
+
+    # Check its 600 not 60
```
Comment (bblommers) [inline tests/test_elbv2/test_elbv2.py:1183]: This comment seems unrelated

**Review by bblommers — CHANGES_REQUESTED** [review]
Hi @thisjulio, welcome to Moto! I've just got two minor nitpicks - happy to merge once they're resolved

**Comment by thisjulio** [comment]
Thanks @bblommers! Copy and Paste fail 😂

**Review by bblommers — APPROVED** [review]
LGTM! Thanks for the PR @thisjulio
